components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/service/DCRMService.java

```diff
@@ -387,6 +388,10 @@ private OAuthConsumerAppDTO createOAuthApp(ApplicationRegistrationRequest regist
         oAuthConsumerApp.setTokenType(registrationRequest.getTokenType());
         oAuthConsumerApp.setBackChannelLogoutUrl(
                 validateBackchannelLogoutURI(registrationRequest.getBackchannelLogoutUri()));
+        if (!registrationRequest.getAudiences().isEmpty()) {
```

Comment:
Shall we use CollectionUtils.isEmpty here?

Reply:
Fixed